app/controllers/application_controller.rb

```diff
@@ -193,7 +193,7 @@ def remote_user_provided?
   end
 
   def display_layout?
-    return nil if two_pane?
+    return false if two_pane?
```

Comment:
`layout` [changed on Rails 4](http://apidock.com/rails/ActionView/Layouts/ClassMethods/layout) - returning nil as we did before, means to return the default layout, so two-pane won't work. 

We have to return false to ensure the 'edit' methods in two-pane are loaded in the two-pane instead of as a fullpage.